streamz/dataframe/utils.py

```diff
@@ -0,0 +1,38 @@
+from enum import Enum
+
+
+def is_dataframe_like(df):
```

Comment:
Can we replace these with imports from dask.dataframe.utils?

Reply:
Yes, this will be possible with cudf 0.6. Currently, is_series_like fails with cudf. Ref: [rapidsai/cudf/#983](https://github.com/rapidsai/cudf/issues/983)

Reply:
Also, this means non dask streamz DataFrame users are forced to install dask?

Reply:
I suppose that is an issue; I don't suppose there are many non-dask cudf users of streamz... but you are right. A pity to duplicate code, though.